actions/banked-months-updater.vbs

```diff
@@ -0,0 +1,482 @@
+update_wreg = False 'defaulting to false unless there are banked months related updates to make 
+spec_memo = False 
+case_note = False 
+Show_dialog = False 
+
+For i = 0 to Ubound(banked_months_array, 2)
+	If banked_months_array(member_in_error_const, i) <> "" then end_msg = end_msg & banked_months_array(member_in_error_const, i) & vbcr & vbcr 
+	If banked_months_array(used_all_banked_mo_const, i) = False then spec_memo = True 
+	If banked_months_array(member_in_error_const, i) = "" then update_wreg = True 
+	If spec_memo = True or update_wreg = True then case_note = True 
+	If banked_months_array(abawd_status_const, i) = "10" or banked_months_array(abawd_status_const, i) = "13" then show_dialog = True 
+Next 
+
+'script end procedure for anyone who is in error 
+If end_msg <> "" then script_end_procedure_with_error_report(end_msg)  
+'Don't show the dialog if no one is already coded as ABAWD or banked for the initial month
+If Show_dialog = False then script_end_procedure_with_error_report("No members in the EATS Household with MEMB 01 are coded as ABAWD (30/10) or Banked (30/13) for the initial month. The script will now end.")
+
+script_actions = "" 'variable that tells users what's going on in dialog. 
+If update_wreg = True then script_actions = script_actions & VBCR & "* The STAT/WREG panel will be updated for members who have banked months available, or who have exhausted them."
+If spec_memo = True then script_actions = script_actions & VBCR & "* A SPEC/MEMO with required Banked Months and TLR Text will be sent to the resident, AREP, and/or SWKR."
+If case_note = True  then script_actions = script_actions & VBCR & "* A detailed CASE/NOTE will be created about case and member updates."
+
+Dialog1 = ""		    '----------------------------------------------------------------------------------------------------Displaying Main Dialog 
+BeginDialog Dialog1, 0, 0, 550, 385, "Banked Months Evaluation Dialog for ABAWD/TLR's on Case #" & MAXIS_case_number
+  Text 65, 10, 455, 10, "**The following information are for the ABAWD and Banked Months counts/months members in the EATS Household containing Memb 01**"
+  Text 115, 25, 315, 10, "--The information below reflects counts from the initial month/year selected in the initial dialog.--"
+
+For i = 0 to Ubound(banked_months_array, 2)
+	If banked_months_array(abawd_status_const, i) = "10" or banked_months_array(abawd_status_const, i) = "13" then 
+		y_pos = (45 + i * 50)
+		GroupBox 10, y_pos, 530, 45, "MEMB #" & banked_months_array(member_number_const , i) & " " & banked_months_array(member_first_name_const, i) & ", ABAWD/FSET: " & banked_months_array(wreg_status_const, i) & "/" & banked_months_array(abawd_status_const, i)
+  		Text 20, y_pos + 15, 255, 10, "ABAWD Count/Months Used: " & banked_months_array(abawd_mo_count_const, i) & " - " & banked_months_array(abawd_mo_string_const, i)
+  		Text 20, y_pos + 30, 255, 10, "ABAWD Months Evaluation: " & banked_months_array(abawd_month_eval_const, i)
```

Comment:
Depending on the length of banked_months_array(abawd_month_eval_const, i) I found that the group box cut off the text. I can provide a screenshot if that is helpful. May want to increase the size of the group text box?